interop.py

```diff
@@ -444,7 +444,7 @@ def run(self):
         nr_failed = 0
         for server in self._servers:
             for client in self._clients:
-                logging.info(
+                logging.debug(
```

Comment:
Why do you think this one is not essential?

Reply:
Because the only thing it prints compared to the next line is the name of the docker image?

Reply:
You're right. I thought it also included the test case.